cmd/explorer/main.go

```diff
@@ -225,15 +228,15 @@ func main() {
 				for _, epoch := range utils.Config.Indexer.OneTimeExport.Epochs {
 					err := exporter.ExportEpoch(epoch, rpcClient)
 					if err != nil {
-						logrus.Fatal(err)
+						utils.LogError("exporting epoch error", err).Fatal()
```

Comment:
I changed the error messages so that now all of them should be unique but I allowed it if one is an LogError and the other LogFatal since you can still distinguish them by the level.